pkg/conversions/virtualrouter_types_conversion.go

```diff
@@ -260,15 +262,15 @@ func Convert_CRD_TCPRoute_To_SDK_TcpRoute(crdObj *appmesh.TCPRoute,
 	return nil
 }
 
-func Convert_CRD_GRPCRouteMetadataMatchMethod_To_SDK_GrpcRouteMetadataMatchMethod(crdObj *appmesh.GRPCRouteMetadataMatchMethod,
-	sdkObj *appmeshsdk.GrpcRouteMetadataMatchMethod, scope conversion.Scope) error {
+func Convert_CRD_GRPCRouteMetadataMatchMethod_To_SDK_GrpcRouteMetadataMatchMethod(crdObj *appmesh.MatchMethod,
```

Comment:
Why `GRPCRouteMetadataMatchMethod` -> 'MatchMethod`? Is this struct common across protocols and we're sharing the same struct across everything?

Reply:
The struct does have same feilds, but now I reverted that change ,  We have separate structs for HTTP Match and Grpc Match

Reply:
Yes, it's better that way as it keeps us in sync with AppMesh and if these structs deviate in future it will be a much bigger change for us.